group_vars/all/nac2022.yml

```diff
@@ -0,0 +1,2 @@
+---
+timezone: America/Los_Angeles
```

Comment:
Does this now only overwrite the wf one because it gets loaded later? @vmcj what is a better way to do that?

Reply:
I think setting this in the hosts file on the group (probably group[all] would be a better place. We already have that file being gitignored.

Reply:
[WARNING]: Skipping key (timezone) in group (group[all]) as it is not a mapping, it is a <class 'ansible.parsing.yaml.objects.AnsibleUnicode'>
please post yaml :)

Reply:
thinking this in hosts.yml: 
```
all:
  vars:
    timezone: America/New_York
```

Reply:
this was pushed.